printer_zpl2/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+ZPL II Label printing
+=====================
+
+This module extends the **Report to printer** (``base_report_to_printer``)
+module to add a ZPL II label printing feature.
+
+It's done to be used from other modules, or from python code blocks.
```

Comment:
I'm not quite sure what you're going for with this statement, so I can't make any recommendations, but it's unclear IMO. Is it that this is a base module, meant to be used with other modules? That's what I'm thinking from reading the rest of ReadMe?